randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1342,7 +1344,9 @@ Extra - asset_id: collision_camera_018
   > Door to Save Station East
       Trivial
   > Tile Group (POWERBEAM)
-      Spider Magnet or Infinite Bomb Jump (Beginner) or Use Spin Boost
+      Any of the following:
+          Spider Magnet or Infinite Bomb Jump (Beginner) or Use Spin Boost
+          Speed Booster and Disabled Door Lock Randomizer
```

Comment:
Speed is fine here, but doesnt seem like you actually put it in what i suggested of Grapple. IBJ trick also still needs changed to Simple IBJ